### ISSUE TITLE
Allow specified tasks

### ISSUE THREAD

**Issue by guestieng** [ryuta46/vscode-multi-command#39]
In the keybindings.json one can set "runTask" as a command and 
directly pass a specific task as argument like this:
```
{
       "key": "\<key binding\>",
       "command": "workbench.action.tasks.runTask",
       "args": "<task label>"
}
```
Can this be done correspondingly for a "multiCommand" or is it possible to implement this, i.e. such that something like the following worked:
```
"multiCommand.commands": [{
      "command": "<label>",
      "sequence": [
          "workbench.action.tasks.runTask",
          { "args": "<task label>" }
      ]
  }],
```

**Comment by ryuta46** [comment]
@guestieng  If you want to pass arguments in a command sequence, you can do the followings:
```
"multiCommand.commands": [{
      "command": "<label>",
      "sequence": [
          { "command": "workbench.action.tasks.runTask",  "args": "<task label>" }
      ]
  }],
```

Please see [this section](https://github.com/ryuta46/vscode-multi-command#pass-arguments-to-commands) in README.md.

**Comment by guestieng** [comment]
Alright, thanks, that works.
Moreover, is there a way to pass the task label as a parameter like this:
```
multiCommand.commands": [{
      "command": "<label>",
      "sequence": [
          { "command": "workbench.action.tasks.runTask",  "args": "${task_label_var}" }
      ]
  }]
```
Tried it, but ${task_label_var} does not seem to be evaluated this way.

**Comment by ryuta46** [comment]
Where did you define `${task_label_var}` ?
Can you use this kind of variable in keybindings.json without multi-command extension like this?

```
{
       "key": "\<key binding\>",
       "command": "workbench.action.tasks.runTask",
       "args": "${task_label_var}"
}
```

**Comment by guestieng** [comment]
Let's say a task may be labelled like, e.g., ${workspaceFolderBasename} .
In fact, it does not work to use this kind of variable in keybindings.json as you exemplified.
So I guess a workaround via "multi-command" is not possible either?

**Comment by ryuta46** [comment]
You are right.
There is no workaround for resolving labels like `${workspaceFolderBasename}` via "multi-command" now.

I think [this issue in vscode repository](https://github.com/microsoft/vscode/issues/46471) is related.

I investigated a little and found that some of variables in [variable references](https://code.visualstudio.com/docs/editor/variables-reference) can be resolved manually from API for vscode extension.

I will consider to support resolving some of predefined variables in multi-command extension in future work.

**Comment by ryuta46** [comment]
Implemented in 1.6.0.
See [Pass arguments to commands](https://github.com/ryuta46/vscode-multi-command/tree/master#pass-arguments-to-commands) in README.

Please reopen this issue or create new issue if you have any problem about this feature.
Thanks.
